src/coreclr/jit/instrswasm.h

```diff
@@ -32,10 +32,114 @@ INST(i32_load,    "i32.load",    0, IF_MEMARG,  0x28)
 INST(i64_load,    "i64.load",    0, IF_MEMARG,  0x29)
 INST(f32_load,    "f32.load",    0, IF_MEMARG,  0x2a)
 INST(f64_load,    "f64.load",    0, IF_MEMARG,  0x2b)
```

Comment:
```suggestion
INST(f32_load,    "f32.load",    0, IF_MEMARG,  0x2A)
INST(f64_load,    "f64.load",    0, IF_MEMARG,  0x2B)
```
These two definitions are still using lowercased base64-letters, should they be changed as well?